ld/nodemcu.ld

```diff
@@ -5,7 +5,7 @@ MEMORY
   dport0_0_seg :                        org = 0x3FF00000, len = 0x10
   dram0_0_seg :                         org = 0x3FFE8000, len = 0x14000
   iram1_0_seg :                         org = 0x40100000, len = 0x8000
-  irom0_0_seg :                         org = 0x40210000, len = 0xD0000
+  irom0_0_seg :                         org = 0x40210000, len = 0xE0000
```

Comment:
This value is also good enough to build #1231, I just tested. So, if we merge this one first we can finally merge that one as well.